### PR TITLE
Update README add installation instructions

### PR DESCRIPTION
**Pull request by Redsandro**
https://github.com/internetarchive/archive-pdf-tools/issues/41#issuecomment-1119541727

### CODE REVIEW

**Comment by mara004** [comment]
I think that's a helpful addition. One can't expect that anyone is familiar with Python tooling.

**Comment by Redsandro** [comment]
(You may notice a couple more commits. The problem is that I can't get the code blocks to render correctly even though I believe I use the correct markdown. Perhaps someone with merge permission can fix it if it still doesn't render correctly after merge.)

**Comment by MerlijnWajer** [comment]
I think this is helpful, but I think it might also make sense to mention that the tagged releases are available on https://pypi.org/project/archive-pdf-tools/ - which means people can run `pip install archive-pdf-tools==1.4.14` for example.

BTW: I have been working on making the read the docs version better, which is here: https://archive-pdf-tools.readthedocs.io/en/latest/ - it's basically a more comprehensive version of the readme (some of it is copy paste)

I think having an installation chapter there would also be good.

**Comment by Redsandro** [comment]
> it might also make sense to mention that the tagged releases are available on

Sounds good. Would you like me to amend the PR or are you asking for opinions before you amend the PR?

> I have been working on making the read the docs version better, which is here:

Where can this be amended? Is it automatically generated from the `docs/` folder?

**Comment by MerlijnWajer** [comment]
Right, that's correct, that is in `docs/`. And yes, you can add it to the PR here if you're up for it.

**Comment by Redsandro** [comment]
> you can add it to the PR here if you're up for it.

Ok, that's about it. Sorry for the many commits. I didn't know RST, thought it was similar to MD. But now it works.

**Comment by MerlijnWajer** [comment]
Thanks, mind if I squash the commits into one?

**Comment by Redsandro** [comment]
Not at all, do what you want. :slightly_smiling_face:

File: README.rst [hunk 1]
```diff
@@ -66,6 +66,50 @@ For JBIG2 compression:
 * `jbig2enc <https://github.com/agl/jbig2enc>`_ for JBIG2 compression (and PyMuPDF 1.19.0 or higher)
 
 
+Installation
+============
+
+First install dependencies. For example, in Ubuntu::
+
+
```
Comment (mara004) [inline README.rst:74]: Any reason you are always leaving 2 lines empty? I think 1 should be quite sufficient...

File: docs/source/index.rst [hunk 2]
```diff
@@ -61,6 +61,50 @@ For JBIG2 compression:
 * `jbig2enc <https://github.com/agl/jbig2enc>`_ for JBIG2 compression (and PyMuPDF 1.19.0 or higher)
 
 
+Installation
+------------
+
```
Comment (mara004) [inline docs/source/index.rst:66]: I'm not sure if it makes sense to duplicate the whole section. I personally wouldn't do that.

Comment (MerlijnWajer) [reply docs/source/index.rst:66]: I think it's not too bad, if anything, maybe the README ought to refer to the docs for specifics.

File: README.rst (hunk 1 above)
Comment (MerlijnWajer) [reply README.rst:74]: I can fix that up

**Comment by MerlijnWajer** [comment]
I've merged this to master, thanks

File: README.rst (hunk 1 above)
Comment (Redsandro) [reply README.rst:74]: > Any reason

When in Rome, do as the Romans do. :slightly_smiling_face: I looked at the first existing code block I found (under "Usage")

**Pull request closed without merging**